src/components/Filters/Aggregation/AggregationItem/AggregationItem.spec.js

```diff
@@ -27,7 +27,7 @@ const renderComponent = (props, newAggsState, newQueryState) => {
     query: newQueryState,
   };
 
-  render(<AggregationItem {...props} />, {
+  return render(<AggregationItem {...props} />, {
```

Comment:
a return statement isn't needed here.

Reply:
Technically not needed, but `render` does return an object with various items (as seen [here](https://testing-library.com/docs/react-testing-library/api/#render)). But since it's not being used here, I'll taken out the return statement.